package-system/OpenMesh/build_config.json

```diff
@@ -0,0 +1,88 @@
+{
+   "git_url":"https://gitlab.vci.rwth-aachen.de:9000/OpenMesh/OpenMesh.git",
+   "git_tag":"OpenMesh-8.1",
+   "patch_file":"openmesh-o3de.patch",
+   "package_name":"OpenMesh",
+   "package_version":"8.1-rev2",
+   "package_url":"https://www.graphics.rwth-aachen.de/software/openmesh/",
+   "package_license":"BSD-3-Clause",
+   "package_license_file":"LICENSE",
+   "cmake_find_target": "FindOpenMesh.cmake",
+   "Platforms":{
+      "Windows":{
+         "Windows":{
+            "cmake_find_template":"FindOpenMesh.cmake.Windows",
+            "custom_build_cmd": [
+               "build_openmesh_windows.cmd"
+            ],
+            "custom_install_cmd": [
+               "install_openmesh_windows.cmd"
+            ]
+         }
+      },
+      "Darwin":{
+         "Mac":{
+            "cmake_find_template":"FindOpenMesh.cmake.Mac",
+            "cmake_generate_args_debug":[
+               "-DDISABLE_QMAKE_BUILD=ON",
+               "-DBUILD_APPS=OFF",
+               "-DOPENMESH_DOCS=OFF",
+               "-DOPENMESH_BUILD_SHARED=OFF",
+               "-DOPENMESH_BUILD_UNIT_TESTS=OFF",
+               "-DCMAKE_CXX_FLAGS=\"-fPIC -fno-aligned-allocation\"",
+               "-DCMAKE_CXX_STANDARD=17",
+               "-DCMAKE_BUILD_TYPE=Debug"
+            ],
```

Comment:
You will need to also supply the mac toolchain file to make sure the right version of mac is given - see the build scripts for tiff and zlib.